### DISCUSSION TITLE
HTTP streaming ?

### DISCUSSION THREAD

**Discussion by KaKi87** [moneytoo/Player#402] [Q&A] [answered]
Hello,

I'm sorry if this question has already been asked but I searched and didn't find anything related to this.

Can this app do HTTP streaming, if yes, how, if no, is it planned ?

Thanks.

**Comment by moneytoo** [comment, marked as answer]
> Can this app do HTTP streaming, if yes, how

Yes. The app is registered to handle possible streaming links so (depending on the version of Android and user configuration) it should be possible to select Just Player when opening/clicking links in other apps. Alternatively, if the link is available only as a text, use the system share to share it to Just Player to open it.

**Reply by KaKi87** [reply]
I like how you implemented the most natural way possible of doing things. :+1: 

However, a bit of documentation would still be welcome :)

Thanks

**Reply by moneytoo** [reply]
Yeah, this one should probably be in the readme so I will put it in my todo. (There are actually few more features I should document more.)
